### PR TITLE
Refactor tooltip label switch into a util

### LINKED ISSUE #37: Refactor mapping switches into their own utils
See explanation at https://github.com/MLH-Fellowship/scheduling-profiler-prototype/pull/34/files#r447887979. There are likely more similar cases that should be refactored.

### PR DESCRIPTION
**Pull request by kartikcho**
This PR fixes #37 by moving tooltip label switch into a util.

### CODE REVIEW

**Comment by kartikcho** [comment]
I saw some potential areas to reduce other switch cases but not sure it would help that much and is better as is.

One such example,

https://github.com/MLH-Fellowship/scheduling-profiler-prototype/blob/5fe70a3c2f85f83da80bc96dbda621364c07adc9/src/canvas/views/ReactEventsView.js#L86-L110

File: src/util/tooltipLabel.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+export function getReactEventLabel(type) {
```
Comment (taneliang) [inline src/util/tooltipLabel.js]: This file needs Flow types

File: src/util/tooltipLabel.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      break;
+    case 'suspense-rejected':
+      label = 'suspense rejected';
+      break;
+    default:
+      break;
+  }
+  return label;
+}
+export function getReactMeasureLabel(type) {
```
Comment (taneliang) [inline src/util/tooltipLabel.js]: style nit: there should be an empty line between functions

```suggestion

export function getReactMeasureLabel(type) {
```

File: src/util/tooltipLabel.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
+export function getReactEventLabel(type) {
+  let label = null;
```
Comment (taneliang) [inline src/util/tooltipLabel.js]: Can we just return the string instead of setting it to a variable?

**Review by taneliang — COMMENTED** [review]
Is there any reason why the code should be extracted into its own file? Since these functions are only used in one place they should remain in the same file in my opinion.

**Comment by taneliang** [comment]
@kartikcho I think that switch in `ReactEventsView` that you highlighted is fine as is too, but I won't be opposed if you wanted to extract that out into a function. I don't have an opinion on that :)

**Comment by kartikcho** [comment]
> 
> Is there any reason why the code should be extracted into its own file? Since these functions are only used in one place they should remain in the same file in my opinion.

That's a valid point, my initial assumption was going to make one file with functions for all the switch case scenarios but they're better as is. I'll revert

**Review by taneliang — APPROVED** [review]
LG!

**Pull request merged**